authors/apps/articles/serializers.py

```diff
@@ -1,14 +1,15 @@
 from rest_framework import serializers
 from django.db.models import Avg
 from django.core.validators import MinValueValidator, MaxValueValidator
+from rest_framework.validators import UniqueTogetherValidator
```

Comment:
@Ramon3162 This file has been imported on line 10

Reply:
Thanks for that catch brotherman, on it. Won't happen again.

Reply:
Noted, I have made the change already.